2019-11-14 14:14:34,748	influx_updater.py	INFO	Program started...
2019-11-14 14:15:00,857	influx_updater.py	INFO	Program started...
2019-11-14 14:16:35,937	influx_updater.py	INFO	Program started...
2019-11-14 14:20:00,208	influx_updater.py	INFO	Successfully connected to psql...
2019-11-14 14:20:00,269	influx_updater.py	ERROR	relation "sraka" does not exist
LINE 1: SELECT * FROM sraka;
                      ^

2019-11-14 14:20:00,340	influx_updater.py	INFO	Successfully connected to influx...
2019-11-14 14:20:49,893	influx_updater.py	INFO	Program started...
2019-11-14 14:21:00,197	influx_updater.py	INFO	Successfully connected to psql...
2019-11-14 14:21:00,259	influx_updater.py	ERROR	relation "sraka" does not exist
LINE 1: SELECT * FROM sraka;
                      ^

2019-11-14 14:21:00,330	influx_updater.py	INFO	Successfully connected to influx...
2019-11-14 14:22:31,895	influx_updater.py	INFO	Program started...
2019-11-14 14:31:35,028	influx_updater.py	INFO	Program started...
2019-11-14 14:32:00,578	influx_updater.py	ERROR	unterminated quoted string at or near "' sdm630;"
LINE 1: ...e FROM registers WHERE registers.energy_meter_id = ' sdm630;
                                                              ^

2019-11-14 14:32:00,578	influx_updater.py	ERROR	Unusual exception: local variable 'data' referenced before assignment
2019-11-14 14:32:43,624	influx_updater.py	INFO	Program started...
2019-11-14 14:33:02,351	influx_updater.py	ERROR	Unusual exception: 401: {"error":"authorization failed"}

2019-11-14 14:33:41,989	influx_updater.py	INFO	Program started...
2019-11-14 14:34:02,538	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:02,539	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:02,539	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:02,540	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:02,540	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:02,541	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:02,541	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:02,542	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:02,542	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:02,542	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:02,543	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:02,543	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:02,543	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:02,543	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:02,544	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:02,544	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:02,544	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:02,544	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:02,545	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:02,545	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:02,545	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:02,546	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:02,546	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:02,546	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:02,546	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:02,547	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:02,547	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:02,831	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:02,832	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:02,832	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:02,833	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:02,833	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:02,834	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:02,834	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:02,834	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:02,835	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:02,835	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:02,836	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:02,836	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:02,836	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:02,836	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:02,837	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:02,837	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:02,837	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:02,838	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:02,838	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:02,838	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:02,839	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:02,839	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:02,839	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:02,839	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:02,840	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:02,840	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:02,840	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:03,122	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:03,123	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:03,123	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:03,124	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:03,126	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:03,126	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:03,127	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:03,127	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:03,127	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:03,128	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:03,128	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:03,128	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:03,129	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:03,129	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:03,129	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:03,130	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:03,130	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:03,130	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:03,130	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:03,131	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:03,131	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:03,131	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:03,131	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:03,132	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:03,132	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:03,132	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:03,133	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:03,415	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:03,415	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:03,416	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:03,416	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:03,417	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:03,417	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:03,417	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:03,418	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:03,418	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:03,418	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:03,419	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:03,419	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:03,419	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:03,420	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:03,420	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:03,420	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:03,421	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:03,421	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:03,421	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:03,421	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:03,422	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:03,422	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:03,422	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:03,422	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:03,423	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:03,423	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:03,423	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:03,707	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:03,708	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:03,708	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:03,709	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:03,709	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:03,709	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:03,710	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:03,710	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:03,711	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:03,711	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:03,711	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:03,712	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:03,712	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:03,712	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:03,712	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:03,713	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:03,713	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:03,713	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:03,714	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:03,714	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:03,714	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:03,714	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:03,715	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:03,715	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:03,715	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:03,716	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:03,716	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:03,998	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:03,998	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:03,999	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:03,999	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:03,999	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:04,000	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:04,000	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:04,001	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:04,001	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:04,001	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:04,002	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:04,002	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:04,002	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:04,003	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:04,003	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:04,003	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:04,004	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:04,004	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:04,004	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:04,005	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:04,005	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:04,005	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:04,005	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:04,006	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:04,006	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:04,006	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:04,006	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:04,287	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:04,287	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:04,288	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:04,288	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:04,289	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:04,289	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:04,290	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:04,290	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:04,290	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:04,290	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:04,291	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:04,291	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:04,291	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:04,292	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:04,292	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:04,292	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:04,293	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:04,293	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:04,293	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:04,293	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:04,294	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:04,294	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:04,294	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:04,294	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:04,295	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:04,295	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:04,295	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:04,578	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:04,579	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:04,579	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:04,580	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:04,580	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:04,581	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:04,581	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:04,581	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:04,582	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:04,582	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:04,582	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:04,583	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:04,583	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:04,583	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:04,584	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:04,584	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:04,584	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:04,585	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:04,585	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:04,585	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:04,585	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:04,586	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:04,586	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:04,586	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:04,587	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:04,587	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:04,587	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:04,868	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:04,868	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:04,869	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:04,869	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:04,870	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:04,870	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:04,870	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:04,871	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:04,871	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:04,871	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:04,872	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:04,872	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:04,872	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:04,873	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:04,873	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:04,873	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:04,874	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:04,874	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:04,874	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:04,875	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:04,875	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:04,875	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:04,875	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:04,876	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:04,876	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:04,876	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:04,876	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:05,161	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:05,161	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:05,162	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:05,162	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:05,162	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:05,163	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:05,163	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:05,164	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:05,164	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:05,164	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:05,165	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:05,165	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:05,165	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:05,166	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:05,166	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:05,166	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:05,166	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:05,167	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:05,167	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:05,167	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:05,168	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:05,168	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:05,168	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:05,169	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:05,169	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:05,169	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:05,169	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:05,450	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:05,450	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:05,451	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:05,451	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:05,451	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:05,452	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:05,452	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:05,453	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:05,453	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:05,453	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:05,454	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:05,454	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:05,454	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:05,455	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:05,455	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:05,455	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:05,455	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:05,456	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:05,456	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:05,456	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:05,457	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:05,457	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:05,457	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:05,457	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:05,458	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:05,458	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:05,458	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:34:05,739	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 0 (voltageL1)
2019-11-14 14:34:05,740	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 2 (voltageL2)
2019-11-14 14:34:05,740	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 4 (voltageL3)
2019-11-14 14:34:05,741	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 6 (currentL1)
2019-11-14 14:34:05,741	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 8 (currentL2)
2019-11-14 14:34:05,742	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 10 (currentL3)
2019-11-14 14:34:05,742	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 12 (activePowerL1)
2019-11-14 14:34:05,743	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 14:34:05,743	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 16 (activePowerL3)
2019-11-14 14:34:05,743	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 14:34:05,744	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 20 (apparentPowerL2)
2019-11-14 14:34:05,744	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 14:34:05,744	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 24 (reactivePowerL1)
2019-11-14 14:34:05,744	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 26 (reactivePowerL2)
2019-11-14 14:34:05,745	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 28 (reactivePowerL3)
2019-11-14 14:34:05,745	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 30 (phaseFactorL1)
2019-11-14 14:34:05,745	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 32 (phaseFactorL2)
2019-11-14 14:34:05,746	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 34 (phaseFactorL3)
2019-11-14 14:34:05,746	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 56 (apparentPower)
2019-11-14 14:34:05,746	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 60 (reactivePower)
2019-11-14 14:34:05,746	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 62 (phaseFactor)
2019-11-14 14:34:05,747	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 70 (frequency)
2019-11-14 14:34:05,747	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 72 (activeEnergyConsumed)
2019-11-14 14:34:05,747	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:34:05,747	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 14:34:05,748	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 78 (reactiveEnergyProduced)
2019-11-14 14:34:05,748	influx_updater.py	ERROR	Client: 2, Host: 0.0.0.0, Port: 502, Slaveaddress: 1 => ModbusClientException: Error during reading register: 52 (activePower)
2019-11-14 14:35:28,850	influx_updater.py	DEBUG	Configured scheduler...
2019-11-14 14:35:28,851	influx_updater.py	INFO	Program started...
2019-11-14 14:36:00,203	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:36:00,264	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:36:00,265	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:36:00,338	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:36:00,523	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:36:00,584	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:36:02,250	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:36:02,289	influx_updater.py	DEBUG	Updated influx wit data from 178.218.234.100
2019-11-14 14:36:02,289	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:37:00,196	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:37:00,258	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:37:00,259	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:37:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:37:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:37:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:37:02,227	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:37:02,264	influx_updater.py	DEBUG	Updated influx wit data from 178.218.234.100
2019-11-14 14:37:02,264	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:38:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:38:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:38:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:38:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:38:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:38:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:38:02,204	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:38:02,240	influx_updater.py	DEBUG	Updated influx wit data from 178.218.234.100
2019-11-14 14:38:02,241	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:39:00,034	influx_updater.py	ERROR	could not connect to server: Connection refused
	Is the server running on host "34.89.239.19" and accepting
	TCP/IP connections on port 5432?

2019-11-14 14:39:00,035	influx_updater.py	DEBUG	Energy meter list is empty...
2019-11-14 14:39:00,035	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:39:00,100	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:39:00,101	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:40:00,035	influx_updater.py	ERROR	could not connect to server: Connection refused
	Is the server running on host "34.89.239.19" and accepting
	TCP/IP connections on port 5432?

2019-11-14 14:40:00,035	influx_updater.py	DEBUG	Energy meter list is empty...
2019-11-14 14:40:00,036	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:40:00,101	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:40:00,101	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:40:31,444	influx_updater.py	DEBUG	Configured scheduler...
2019-11-14 14:40:31,445	influx_updater.py	INFO	Program started...
2019-11-14 14:41:00,037	influx_updater.py	ERROR	could not connect to server: Connection refused
	Is the server running on host "34.89.239.19" and accepting
	TCP/IP connections on port 5432?

2019-11-14 14:41:00,038	influx_updater.py	DEBUG	Energy meter list is empty...
2019-11-14 14:41:00,109	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:41:00,109	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:42:00,196	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:42:00,259	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:42:00,259	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:42:00,323	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:42:00,514	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:42:00,576	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:42:02,225	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:42:02,264	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:42:02,264	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:43:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:43:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:43:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:43:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:43:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:43:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:43:02,245	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:43:02,283	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:43:02,283	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:44:00,195	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:44:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:44:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:44:00,356	influx_updater.py	ERROR	HTTPConnectionPool(host='34.89.239.19', port=8086): Max retries exceeded with url: /ping (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10183c350>: Failed to establish a new connection: [Errno 61] Connection refused'))
2019-11-14 14:44:00,544	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:44:00,605	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:44:02,250	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:44:02,418	influx_updater.py	ERROR	Unusual exception: HTTPConnectionPool(host='34.89.239.19', port=8086): Max retries exceeded with url: /write?db=test (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x101847450>: Failed to establish a new connection: [Errno 61] Connection refused'))
2019-11-14 14:45:00,220	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:45:00,281	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:45:00,282	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:45:00,383	influx_updater.py	ERROR	HTTPConnectionPool(host='34.89.239.19', port=8086): Max retries exceeded with url: /ping (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x101853dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
2019-11-14 14:45:00,571	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:45:00,633	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:45:03,082	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 14:45:03,284	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:45:03,485	influx_updater.py	ERROR	Unusual exception: HTTPConnectionPool(host='34.89.239.19', port=8086): Max retries exceeded with url: /write?db=test (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10183cb10>: Failed to establish a new connection: [Errno 61] Connection refused'))
2019-11-14 14:46:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:46:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:46:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:46:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:46:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:46:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:46:02,213	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:46:02,337	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:46:02,337	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:47:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:47:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:47:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:47:00,316	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:47:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:47:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:47:02,200	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:47:02,240	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:47:02,241	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:48:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:48:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:48:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:48:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:48:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:48:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:48:02,205	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:48:02,244	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:48:02,244	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:49:00,189	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:49:00,250	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:49:00,250	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:49:00,315	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:49:00,501	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:49:00,562	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:49:02,206	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:49:02,245	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:49:02,245	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:50:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:50:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:50:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:50:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:50:00,511	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:50:00,573	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:50:02,246	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:50:02,285	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:50:02,285	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:51:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:51:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:51:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:51:00,317	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:51:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:51:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:51:02,222	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:51:02,260	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:51:02,261	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:52:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:52:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:52:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:52:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:52:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:52:00,567	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:52:02,207	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:52:02,245	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:52:02,245	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:53:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:53:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:53:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:53:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:53:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:53:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:53:02,550	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:53:02,588	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:53:02,589	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:54:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:54:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:54:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:54:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:54:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:54:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:54:02,207	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:54:02,245	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:54:02,245	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:55:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:55:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:55:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:55:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:55:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:55:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:55:02,292	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:55:02,331	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:55:02,331	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:56:00,195	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:56:00,257	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:56:00,258	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:56:00,323	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:56:00,512	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:56:00,575	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:56:02,200	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:56:02,239	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:56:02,240	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:57:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:57:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:57:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:57:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:57:00,504	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:57:00,566	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:57:02,210	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:57:02,249	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:57:02,249	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:58:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:58:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:58:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:58:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:58:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:58:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:58:02,196	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:58:02,234	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:58:02,235	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 14:59:00,188	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:59:00,250	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:59:00,250	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 14:59:00,315	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 14:59:00,502	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 14:59:00,564	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 14:59:02,267	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 14:59:02,305	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 14:59:02,306	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:00:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:00:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:00:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:00:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:00:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:00:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:00:02,210	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:00:02,250	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:00:02,250	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:01:00,197	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:01:00,257	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:01:00,258	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:01:00,323	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:01:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:01:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:01:02,217	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:01:02,255	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:01:02,255	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:02:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:02:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:02:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:02:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:02:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:02:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:02:02,195	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:02:02,233	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:02:02,233	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:03:00,200	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:03:00,261	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:03:00,262	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:03:00,327	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:03:00,516	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:03:00,578	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:03:02,222	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:03:02,261	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:03:02,262	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:04:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:04:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:04:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:04:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:04:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:04:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:04:02,216	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:04:02,254	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:04:02,255	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:05:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:05:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:05:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:05:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:05:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:05:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:05:02,200	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:05:02,238	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:05:02,238	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:06:00,196	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:06:00,258	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:06:00,259	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:06:00,324	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:06:00,512	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:06:00,574	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:06:02,231	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:06:02,270	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:06:02,270	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:07:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:07:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:07:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:07:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:07:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:07:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:07:02,214	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:07:02,253	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:07:02,253	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:08:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:08:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:08:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:08:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:08:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:08:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:08:02,222	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:08:02,260	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:08:02,261	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:09:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:09:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:09:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:09:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:09:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:09:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:09:02,243	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:09:02,281	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:09:02,282	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:10:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:10:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:10:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:10:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:10:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:10:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:10:02,301	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:10:02,340	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:10:02,340	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:11:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:11:00,257	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:11:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:11:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:11:00,513	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:11:00,575	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:11:02,198	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:11:02,236	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:11:02,237	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:12:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:12:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:12:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:12:00,317	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:12:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:12:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:12:04,911	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 22 (apparentPowerL3)
2019-11-14 15:12:09,133	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:12:09,172	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:12:09,172	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:13:00,190	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:13:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:13:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:13:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:13:00,504	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:13:00,565	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:13:02,235	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:13:02,275	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:13:02,275	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:14:00,195	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:14:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:14:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:14:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:14:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:14:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:14:02,241	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:14:02,281	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:14:02,281	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:15:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:15:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:15:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:15:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:15:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:15:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:15:02,437	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:15:02,476	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:15:02,476	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:16:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:16:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:16:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:16:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:16:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:16:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:16:02,285	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:16:02,324	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:16:02,324	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:17:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:17:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:17:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:17:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:17:00,505	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:17:00,566	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:17:02,195	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 14 (activePowerL2)
2019-11-14 15:17:03,508	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:17:03,547	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:17:03,547	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:18:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:18:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:18:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:18:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:18:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:18:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:18:02,374	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:18:02,413	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:18:02,414	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:19:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:19:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:19:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:19:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:19:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:19:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:19:02,257	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:19:02,296	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:19:02,297	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:20:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:20:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:20:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:20:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:20:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:20:00,567	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:20:02,341	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:20:02,380	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:20:02,380	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:21:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:21:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:21:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:21:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:21:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:21:00,571	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:21:02,678	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:21:02,717	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:21:02,717	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:22:00,195	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:22:00,257	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:22:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:22:00,322	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:22:00,512	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:22:00,574	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:22:02,211	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:22:02,250	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:22:02,250	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:23:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:23:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:23:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:23:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:23:00,512	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:23:00,575	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:23:02,211	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:23:02,249	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:23:02,250	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:24:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:24:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:24:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:24:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:24:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:24:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:24:02,251	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:24:02,289	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:24:02,290	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:25:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:25:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:25:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:25:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:25:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:25:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:25:02,132	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 18 (apparentPowerL1)
2019-11-14 15:25:03,938	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 74 (activeEnergyProduced)
2019-11-14 15:25:04,140	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:25:04,178	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:25:04,179	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:26:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:26:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:26:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:26:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:26:00,512	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:26:00,574	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:26:03,031	influx_updater.py	ERROR	Client: 1, Host: 178.218.234.100, Port: 2137, Slaveaddress: 1 => ModbusClientException: Error during reading register: 76 (reactiveEnergyConsumed)
2019-11-14 15:26:03,166	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:26:03,207	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:26:03,207	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:27:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:27:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:27:00,252	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:27:00,315	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:27:00,502	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:27:00,564	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:27:02,200	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:27:02,239	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:27:02,239	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:28:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:28:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:28:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:28:00,320	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:28:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:28:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:28:02,239	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:28:02,277	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:28:02,278	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:29:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:29:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:29:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:29:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:29:00,505	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:29:00,566	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:29:02,231	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:29:02,269	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:29:02,270	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:30:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:30:00,254	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:30:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:30:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:30:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:30:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:30:02,262	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:30:02,301	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:30:02,301	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:31:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:31:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:31:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:31:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:31:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:31:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:31:02,223	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:31:02,261	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:31:02,261	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:32:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:32:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:32:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:32:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:32:00,506	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:32:00,568	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:32:02,204	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:32:02,243	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:32:02,243	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:33:00,192	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:33:00,253	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:33:00,254	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:33:00,319	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:33:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:33:00,572	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:33:02,222	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:33:02,261	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:33:02,261	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:34:00,193	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:34:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:34:00,255	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:34:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:34:00,511	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:34:00,574	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:34:02,241	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:34:02,280	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:34:02,280	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:35:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:35:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:35:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:35:00,322	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:35:00,513	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:35:00,574	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:35:02,278	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:35:02,318	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:35:02,318	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:36:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:36:00,255	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:36:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:36:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:36:00,510	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:36:00,573	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:36:02,198	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:36:02,237	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:36:02,238	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:37:00,196	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:37:00,258	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:37:00,258	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:37:00,324	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:37:00,511	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:37:00,573	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:37:02,278	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:37:02,317	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:37:02,317	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:38:00,196	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:38:00,258	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:38:00,258	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:38:00,322	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:38:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:38:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:38:02,184	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:38:02,222	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:38:02,223	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:39:00,188	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:39:00,249	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:39:00,249	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:39:00,315	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:39:00,502	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:39:00,563	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:39:02,319	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:39:02,358	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:39:02,358	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:40:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:40:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:40:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:40:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:40:00,505	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:40:00,567	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:40:02,287	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:40:02,325	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:40:02,326	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:41:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:41:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:41:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:41:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:41:00,508	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:41:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:41:02,215	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:41:02,254	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:41:02,254	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:42:00,195	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:42:00,257	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:42:00,257	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:42:00,323	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:42:00,511	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:42:00,573	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:42:02,214	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:42:02,252	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:42:02,253	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:43:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:43:00,263	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:43:00,263	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:43:00,328	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:43:00,515	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:43:00,578	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:43:02,215	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:43:02,253	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:43:02,253	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:44:00,190	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:44:00,251	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:44:00,251	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:44:00,316	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:44:00,503	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:44:00,564	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:44:02,215	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:44:02,254	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:44:02,254	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:45:00,197	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:45:00,259	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:45:00,260	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:45:00,325	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:45:00,514	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:45:00,576	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:45:02,329	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:45:02,369	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:45:02,369	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:46:00,194	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:46:00,256	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:46:00,256	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:46:00,321	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:46:00,509	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:46:00,570	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:46:02,234	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:46:02,273	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:46:02,274	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:47:00,189	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:47:00,250	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:47:00,250	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:47:00,315	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:47:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:47:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:47:02,244	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:47:02,281	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:47:02,282	influx_updater.py	DEBUG	Successfully disconnected from influx...
2019-11-14 15:48:00,191	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:48:00,252	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:48:00,253	influx_updater.py	DEBUG	Updated energy meter list...
2019-11-14 15:48:00,318	influx_updater.py	DEBUG	Successfully connected to influx...
2019-11-14 15:48:00,507	influx_updater.py	DEBUG	Successfully connected to psql...
2019-11-14 15:48:00,569	influx_updater.py	DEBUG	Successfully disconnected from psql...
2019-11-14 15:48:02,232	influx_updater.py	DEBUG	Got values from 178.218.234.100
2019-11-14 15:48:02,270	influx_updater.py	DEBUG	Updated influx with data from 178.218.234.100
2019-11-14 15:48:02,271	influx_updater.py	DEBUG	Successfully disconnected from influx...
